pgmpy/estimators/GES.py

```diff
@@ -55,6 +55,20 @@ def __init__(self, data, use_cache=True, **kwargs):
 
         super(GES, self).__init__(data=data, **kwargs)
 
+    def Test_type(self):#Ideally I would run this in over to choose the proper test depending on the kind of data.
```

Comment:
1. We use snake case in pgmpy, so you could name the function as `test_type`. 
2. I think it would be nice to have a more informative name something such as `get_scoring_method`
3. I think it would be better to move the method to `StructureScore.py`. We can then import the method in all the algorithm files. This would avoid duplicated code.